scripts/maasutils.py

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env python
+import click
+import pyrax
+import requests
+
+
+@click.option("--username", required=True)
+@click.option("--api-key", required=True)
```

Comment:
--help will show the options. While required options is an oxymoron, its not entirely frowned upon. Some people think that arguments should be used for all required params. Other people use options for everything because it means the order of params is not important and gives a name to each parameter that is supplied. I'm in the second camp..